lib/heroku/command/domains.rb

```diff
@@ -13,17 +14,35 @@ class Domains < Base
     #Examples:
     #
     # $ heroku domains
-    # === Domain names for example
-    # example.com
+    # === Development Domain
+    # example.herokuapp.com
+    #
+    # === Custom Domains
+    # Domain Name  CNAME Target
```

Comment:
But `CNAME` implies using that DNS record type, which you shouldn't in all cases. `Record Target` or something more generic seems more sensible to me.